packages/preact/src/index.ts

```diff
@@ -356,7 +356,7 @@ export function useComputed<T>(compute: () => T) {
 	return useMemo(() => computed<T>(() => $compute.current()), []);
 }
 
-export function useSignalEffect(cb: () => void | (() => void)) {
+export function useSignalEffect(cb: () => unknown | (() => unknown)) {
```

Comment:
What's the type of the callback passed to `effect`? Should this type match that since this callback is invoked inside of an `effect` and its return value is passed to `effect`? IIRC, `effect` accepts a clean up function

Reply:
Hmm, good point.  We could make the change from
```ts

	useEffect(() => {
		return effect(() => callback.current());
}, []);
```
to
```

	useEffect(() => {
		return effect(() => { callback.current() });
}, []);
```
This would ensure that `effect` is getting a void returning function.  Alternatively, we could change the definition of `effect`, but I think that is in a different repository?

Reply:
I went ahead and made the above change to this PR, can revert if another solution is desired.